src/feature/stats/rephist.c

```diff
@@ -1527,6 +1533,52 @@ rep_hist_load_state(or_state_t *state, char **err)
   return 0;
 }
 
+/** Collects BandwidthStatistics parameter from consensus.
+ *  Returns 0 iff BandwidthStatistics is 0 or
+ *  the consensus param is 0. */
+bool
+get_bandwidth_stats_param(void)
+{
+  int bandwidth_stats = get_options()->BandwidthStatistics;
+  switch (bandwidth_stats) {
```

Comment:
We avoid switch statements when there are only 2-3 options. "Not equals -1" (-1 means auto) is simpler here.